### PR TITLE
Feature/logger aggregation

### PR DESCRIPTION
**Pull request by Mkl3364**

### CODE REVIEW

File: client/src/app/barathon/create/page.tsx [hunk 1]
```diff
@@ -39,18 +39,17 @@ const CreateBarathon = () => {
...
-        barSelected.filter(
-          (bar) => bar !== (e.target.value as unknown as BarType),
-        ),
-      )
+      setBarSelected(barSelected.filter((bar) => bar.bar.id !== bar.id))
     }
   }
 
+
+  console.log(barSelected)
```
Comment (LeoPhilip33) [inline client/src/app/barathon/create/page.tsx:51]: a virer

File: client/src/app/profile/barathon/[barathonId]/page.tsx [hunk 2]
```diff
@@ -49,19 +49,21 @@ const BarathonIdPage = ({ params }: { params: { barathonId: string } }) => {
...
-        token: user.token,
-      }),
-  })
+  console.log(barathonBars)
+
+  // const { data: bars } = useQuery({
+  //   queryKey: ['bars'],
+  //   queryFn: () =>
+  //     getBars({
+  //       latitude: longitude,
```
Comment (LeoPhilip33) [inline client/src/app/profile/barathon/[barathonId]/page.tsx:61]: a virer

File: client/src/utils/logger.ts [hunk 3]
```diff
@@ -1,14 +1,28 @@
 import log from 'loglevel'
+import * as Sentry from '@sentry/nextjs'
 
 const logger = log.getLogger('NextJSApp')
 log.setLevel('info') // Set the default log level
 
+
+// Sentry.init({
+//   dsn: 'https://80e2c68e14cba2bceb53a94315877a98@o4506186000498688.ingest.sentry.io/4506186003382272',
+//   // https://80e2c68e14cba2bceb53a94315877a98@o4506186000498688.ingest.sentry.io/4506186003382272
```
Comment (LeoPhilip33) [inline client/src/utils/logger.ts:10]: a virer

File: client/src/utils/logger.ts [hunk 4]
```diff
@@ -1,14 +1,28 @@
...
+// });
+
 log.methodFactory = function (methodName, logLevel, loggerName) {
   const rawMethod = log.methodFactory(methodName, logLevel, loggerName)
   return function (message) {
     const timestamp = new Date().toISOString()
     rawMethod(`[${timestamp}] [${methodName.toUpperCase()}] ${message}`)
+    if (methodName === 'error') {
+      Sentry.captureMessage(message);
+      // You can also use Sentry.captureException() for capturing exceptions
```
Comment (LeoPhilip33) [inline client/src/utils/logger.ts:20]: a virer

**Comment by YohannTonnerre** [comment]
aze ca part

**Pull request merged**